### PR TITLE
Update analyzer docs

### PR DESCRIPTION
**Pull request by areed**

### CODE REVIEW

File: docs/gatsby-config.js [hunk 1]
```diff
@@ -72,7 +72,6 @@ const themeOptions = {
               'analyze/deployment-status',
               'analyze/statefulset-status',
               'analyze/image-pull-secrets',
-              'analyze/ingress',
```
Comment (markpundsack) [inline docs/gatsby-config.js:75]: Why is ingress deleted? Is it not an actual analyzer?

File: docs/source/analyze/image-pull-secrets.md [hunk 2]
```diff
@@ -3,23 +3,28 @@ title: Image Pull Secrets
...
+    - imagePullSecret:
+        checkName: Pull from Quay
+        registryName: quay.io
         outcomes:
           - fail:
-              message: You don't have a pg uri secret
+              message: You won't be able to pull from Quay
           - pass:
-              message: Probably a green light connecting to pg
+              message: Images can be pulled from Quay
```
Comment (markpundsack) [inline docs/source/analyze/image-pull-secrets.md:27]: Does this actually test the ability to pull from quay.io, or just test that there's a secret for that registry? The secret could be invalid.

**Comment by markpundsack** [comment]
@areed Lots of good catches in there! Thanks for going through it all.

File: docs/gatsby-config.js (hunk 1 above)
Comment (areed) [reply docs/gatsby-config.js:75]: It doesn't work at all https://github.com/replicatedhq/troubleshoot/issues/264. The [code](https://github.com/replicatedhq/troubleshoot/blob/master/pkg/analyze/ingress.go#L12) is looking at storage classes.

File: docs/source/analyze/image-pull-secrets.md (hunk 2 above)
Comment (areed) [reply docs/source/analyze/image-pull-secrets.md:27]: Only that a secret exists for that registry.

File: docs/gatsby-config.js (hunk 1 above)
Comment (markpundsack) [reply docs/gatsby-config.js:75]: Oh man, that's rough. I'm torn then on removing documentation vs, say either fixing or removing the code.

File: docs/source/analyze/image-pull-secrets.md [hunk 3]
```diff
@@ -3,23 +3,31 @@ title: Image Pull Secrets
...
+    - imagePullSecret:
+        checkName: Pull from Quay
+        registryName: quay.io
         outcomes:
           - fail:
-              message: You don't have a pg uri secret
+              message: You won't be able to pull from Quay
           - pass:
-              message: Probably a green light connecting to pg
+              message: Images can probably be pulled from Quay
```
Comment (markpundsack) [inline docs/source/analyze/image-pull-secrets.md:30]:
```suggestion
              message: Found credentials to pull from Quay
```

File: docs/source/analyze/image-pull-secrets.md [hunk 4]
```diff
@@ -3,23 +3,31 @@ title: Image Pull Secrets
...
-        secretName: postgres
-        namespace: default
-        key: uri
+    - imagePullSecret:
+        checkName: Pull from Quay
+        registryName: quay.io
         outcomes:
           - fail:
-              message: You don't have a pg uri secret
+              message: You won't be able to pull from Quay
```
Comment (markpundsack) [inline docs/source/analyze/image-pull-secrets.md:28]:
```suggestion
              message: Did not find credentials to pull from Quay
```

**Review by markpundsack — APPROVED** [review]

**Pull request merged**